test/Xunit.Xml.TestLogger.NetCore.Tests/expectedLogFile.xml

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assemblies timestamp="05/20/2018 12:27:22">
+  <assembly name="C:\Users\samadala\src\public\xunit.testlogger\test\Xunit.Xml.TestLogger.NetCore.Tests\bin\Debug\netcoreapp1.0\Xunit.Xml.TestLogger.NetCore.Tests.dll" run-date="2018-05-20" run-time="12:27:22" total="4" passed="2" failed="2" skipped="0" time="0.038" errors="0">
```

Comment:
The paths will not allow us to compare file as it is. We may need to compare specific parts of the xml.

Reply:
We should write these assertions in C# if possible.

Reply:
This file is just for reference/doc. Not using this in tests.